src/components/Carousel/ControlledCarousel.js

```diff
@@ -0,0 +1,195 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import { IconButton } from '../Button';
+
+class ControlledCarousel extends Component {
+  onClickIndicator = (e) => {
+    const { onPanelChange } = this.props;
+
+    e.stopPropagation();
+    e.target.blur();
+    onPanelChange(Number(e.target.dataset.index));
+  }
+
+  onKeyboardInteraction = (e) => {
+    if (e.keyCode === 37) {
```

Comment:
Yuu could use the standard `key` property instead, see: [Spec](https://www.w3.org/TR/uievents-key/#named-key-attribute-values)